assets/js/components/notifications/SwitchedToGA4Banner.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
```

Comment:
Usually, we put very dumb file descriptions which are 100% useless. There is no need to add something like `ComponentA component` since it's already clear that the `ComponentA.js` file that contains exportable functional component `ComponentA` is for that component. It makes sense to add it only if we want to explain something really useful.

Reply:
Understood, thanks!